operations/resiliency/Resiliency_Testing_Procedure.md

```diff
@@ -1,240 +1,386 @@
+      ```bash
+      ncn# kubectl label node ncn-w001 uas=False --overwrite
+      ncn# kubectl label node ncn-w003 uas=False --overwrite
+      ```
+
+      If successful, output similar to `node/ncn-w00X labeled` will be returned.
+
+   1. Verify the labels were applied successfully.
+
+      ```bash
+      ncn# kubectl get nodes -l uas=False
+      ```
+
+      Example output:
+
+      ```
+      NAME       STATUS   ROLES    AGE     VERSION
+      ncn-w001   Ready    <none>   4d19h   v1.18.2
+      ncn-w003   Ready    <none>   4d19h   v1.18.2
+      ```
    
-   To create a UAI, see the `Validate UAI Creation` section of [Validate CSM Health](../validate_csm_health.md) and/or [Create a UAI](../UAS_user_and_admin_topics/Create_a_UAI.md).
-   To remove the labels set above after the UAI has been created, run the following:
+   1. Remove the labels set above after the UAI has been created.
+
+      > **IMPORTANT:** After a UAI has been created, labels must be cleared or else when Kubernetes terminates the running UAI on the NCN being shut down, it will not be able to reschedule the UAI on another pod.
+      
+      ```bash
+      ncn# kubectl label node ncn-w001 uas-
+      ncn# kubectl label node ncn-w003 uas-
+      ```
+
+1. Verify that WLM is configured with the appropriate workload manager within the created UAI.
    
-   ```bash
-   ncn# kubectl label node ncn-w001 uas-
-   ncn# kubectl label node ncn-w003 uas-
-   ```
-* Within the created UAI, verify that WLM is configured with the appropriate workload manager.
-  Verify the connection string of the created UAI:
+   1. Verify the connection string of the created UAI.
    
-   ```bash
-   ncn# cray uas list
-   ```
+      ```bash
+      ncn# cray uas list
+      ```
 
-   Example output:
+      Example output:
 
-   ```
-   [[results]]
-   username =  "uastest"
-   uai_host =  "ncn-w001"
-   uai_status =  "Running: Ready"
-   uai_connect_string = "ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa"
-   uai_img =  "bis.local:5000/cray/cray-uas-sles15sp1-slurm:latest"
-   uai_age =  "1m"
-   uai_name =  "uai-uastest-5653e9b9"
-   ```
-   Login to the created UAI (example):
+      ```
+      [[results]]
+      username =  "uastest"
+      uai_host =  "ncn-w001"
+      uai_status =  "Running: Ready"
+      uai_connect_string = "ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa"
+      uai_img =  "bis.local:5000/cray/cray-uas-sles15sp1-slurm:latest"
+      uai_age =  "1m"
+      uai_name =  "uai-uastest-5653e9b9"
+      ```
    
-   ```bash
-   ncn# ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa
-   ```
-   To verify the configuration of slurm, for example, within the UAI:
+   1. Log in to the created UAI.
+
+      For example:
+
+      ```bash
+      ncn# ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa
+      ```
    
+   1. Verify the configuration of Slurm, for example, within the UAI:
+   
+      ```bash
+      uastest@uai-uastest-5653e9b9:/lus/uastest> srun -N 4 hostname | sort
```

Comment:
Replace the "uastest@uai-uastest-5653e9b9:/lus/uastest> srun -N 4 hostname | sort"
with, to match the UAI above:

vers@uai-vers-f8fa541f-5c5c9f5b75-vsb2k:/lus> srun -N 4 hostname | sort

Reply:
fixed